01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,24 @@
+const {readFileSync, writeFileSync} = require("fs");
+
+const firstLine = writeFileSync('./temporary/fileA.txt', 
+'We used writeFileSync from fs module to write some text into fileA.txt \n',
+);
+
+const secondLine = writeFileSync('./temporary/fileA.txt', 
+'Second line for example \n',
+{flag: 'a'}
+);
+
+const thirdLine = writeFileSync('./temporary/fileA.txt', 
+'Third line for example \n',
+{flag: 'a'}
+);
+
+
+const firstFile = readFileSync('./temporary/fileA.txt', 'utf8');
+const secondFile = readFileSync('./temporary/fileA.txt', 'utf8');
```

Comment:
It's the same file so no need to read it multiple times

Reply:
Ok Mr Zach. I understood it. Thank for reviewing my code.